src/three/plugins/SUBTREELoader.js

```diff
@@ -659,87 +658,74 @@ export class SUBTREELoader extends LoaderBase {
-			const maxY = boundingVolume[ 3 ];
-			const sizeX = ( maxX - minX ) / ( Math.pow( 2, tile.__level ) );
-			const sizeY = ( maxY - minY ) / ( Math.pow( 2, tile.__level ) );
-			boundingVolume[ 0 ] = minX + sizeX * tile.__x;	//west
-			boundingVolume[ 2 ] = minX + sizeX * ( tile.__x + 1 );	//east
-			boundingVolume[ 1 ] = minY + sizeY * tile.__y;	//south
-			boundingVolume[ 3 ] = minY + sizeY * ( tile.__y + 1 );	//north
+			const region = [ ...this.rootTile.boundingVolume.region ];
+			const minX = region[ 0 ];
+			const maxX = region[ 2 ];
+			const minY = region[ 1 ];
+			const maxY = region[ 3 ];
+			const sizeX = ( maxX - minX ) / Math.pow( 2, tile.__level );
+			const sizeY = ( maxY - minY ) / Math.pow( 2, tile.__level );
+			region[ 0 ] = minX + sizeX * tile.__x;	//west
+			region[ 2 ] = minX + sizeX * ( tile.__x + 1 );	//east
+			region[ 1 ] = minY + sizeY * tile.__y;	//south
+			region[ 3 ] = minY + sizeY * ( tile.__y + 1 );	//north
 
 			for ( let k = 0; k < 4; k ++ ) {
 
-				const coord = boundingVolume[ k ];
-				if ( coord < - Math.PI ) boundingVolume[ k ] += 2 * Math.PI; else if ( coord > Math.PI ) boundingVolume[ k ] -= 2 * Math.PI;
+				const coord = region[ k ];
+				if ( coord < - Math.PI ) {
 
-			}
-			return { region: boundingVolume };
+					region[ k ] += 2 * Math.PI;
 
-		} else if ( this.rootTile.boundingVolume.box ) {
+				} else if ( coord > Math.PI ) {
 
-			/*
-			An array of 12 numbers that define an oriented bounding box.
-			 The first three elements define the x, y, and z values for the center of the box.
-			 The next three elements (with indices 3, 4, and 5) define the x axis direction and half-length.
-			  The next three elements (indices 6, 7, and 8) define the y axis direction and half-length.
-			 The last three elements (indices 9, 10, and 11) define the z axis direction and half-length.
-			 */
+					region[ k ] -= 2 * Math.PI;
 
-			boundingVolume = [ ...this.rootTile.boundingVolume.box ];
-			const scale = Math.pow( 2, - tile.__level );
+				}
 
-			const vecCenter = new Vector3();
-			vecCenter.fromArray( boundingVolume.slice( 0, 3 ) );
+			}
+
+			boundingVolume.region = region;
 
-			const modelSpaceX = - 1 + ( 2 * tile.__x + 1 ) * scale;
-			const modelSpaceY = - 1 + ( 2 * tile.__y + 1 ) * scale;
-			let modelSpaceZ = 0;
+		}
 
-			let rootHalfAxes = new Matrix3();
-			rootHalfAxes.fromArray( boundingVolume, 3 );
-			const scaleFactors = new Vector3( scale, scale, 1 );
-			if ( ! isNaN( tile.__z ) ) {
+		if ( this.rootTile.boundingVolume.box ) {
 
-				modelSpaceZ = - 1 + ( 2 * tile.__z + 1 ) * scale;
-				scaleFactors.z = scale;
+			// 0-2: center of the box
+			// 3-5: x axis direction and half length
+			// 6-8: y axis direction and half length
+			// 9-11: z axis direction and half length
+
+			const box = [ ...this.rootTile.boundingVolume.box ];
+			const cellSteps = 2 ** tile.__level - 1;
+			const scale = Math.pow( 2, - tile.__level );
+			for ( let i = 0; i < 3; i ++ ) {
```

Comment:
Should it not  stop at 2 and leave Z untouched ?
```suggestion
			for ( let i = 0; i < 2; i ++ ) {
```

Reply:
Thanks for the catch - I was ignoring the z component of each axis rather than the z-axis of the bounding box.